### ISSUE TITLE
Accessibility: unlabeled buttons when using Talkback screen reader

### ISSUE THREAD

**Issue by shortwavesurfer2009** [m2049r/xmrwallet#885]
All these issues are experienced when using the talkback screen reader which can be enabled in android settings > Accessibility

- [x] On the wallet selection screen the settings icon at the top left reads as "unlabeled"
- [x] on the wallet selection screen the "+" icon at the bottom right reads as "unlabeled"
- [x] on the wallet selection screen the "x" button to the right of "Ledger enabled, tap for more info." reads as "unlabeled"
- [x] on the wallet selection the globe icon to enable tor to the left of "opennode.xmr-tw.org" reads as "unlabeled"
- [x] on the wallet selection screen the refresh icon to the right of "opennode.xmr-tw.org" reads as "unlabeled"
- [ ] on the wallet select screen when the "+" is pressed the menu options for "create new wallet", "restore view-only wallet", "restore wallet from private keys", "restore wallet 25 word seed", "import wallet", and the "x" cannot be focused by talkback at all and cannot be used.

When a wallet is open
- [ ] the three line hamburger menu at the top left reads as "unlabeled"

On the "receive screen
- [x] the back arrow at the top left reads " unlabeled"
- [x] the copy to clipboard icon to the right of the monero address reads as "unlabeled"
- [x] the qr code button reads as "unlabeled"

On the "give screen
- [x] the " x" at the top left reads as "unlabeled"
- [x] the paste icon to the right of receiver address box reads as "unlabeled"
- [ ] the row of buttons for monero, bitcoin, litecoin, etherium, dash, and doge all read as "unlabeled"

Please be aware, these are all issues discovered at first glance. There are likely many more of these i have not seen yet

**Comment by shortwavesurfer2009** [comment]
Hey, I just went through these issues and there are still some problems so this needs to be reopened. I checked everything that is fixed and left the things that needs still to be fixed unchecked.
